# HG changeset patch
# User Rik <rik@octave.org>
# Date 1411445573 25200
#      Mon Sep 22 21:12:53 2014 -0700
# Node ID 431dc1da050c04e65af43f968ee2df3bf9d2afc3
# Parent  0f9c5a15c8faa5f06ce482188e33134d01609b22
doc: Periodic spellcheck of documentation.

* aspell-octave.en.pws: Add new words to Octave's exception dictionary.

* ichol.m, ilu.m, lscov.m: Use @nospell macro around certain words.

diff --git a/doc/interpreter/doccheck/aspell-octave.en.pws b/doc/interpreter/doccheck/aspell-octave.en.pws
--- a/doc/interpreter/doccheck/aspell-octave.en.pws
+++ b/doc/interpreter/doccheck/aspell-octave.en.pws
@@ -883,16 +883,17 @@ spstats
 sqrt
 sqrtm
 sr
 ss
 stairstep
 startup
 Startup
 statinfo
+stdin
 stdnormal
 stdout
 stepsize
 STFT
 STL
 stochastically
 strcmp
 stringanchors
@@ -1108,16 +1109,17 @@ xerrorbars
 XFig
 xGBTRF
 xGELSD
 xGETRF
 xGTSV
 xl
 xmax
 xmin
+xon
 xPBTRF
 xPOTRF
 xPTSV
 xtest
 xtests
 xTRTRS
 xu
 xwd
diff --git a/scripts/sparse/ichol.m b/scripts/sparse/ichol.m
--- a/scripts/sparse/ichol.m
+++ b/scripts/sparse/ichol.m
@@ -36,24 +36,26 @@
 ## is the value of field.  Names and specifiers are case sensitive.
 ##
 ## @table @asis
 ## @item type
 ## Type of factorization.
 ## String indicating which flavor of incomplete Cholesky to perform.  Valid
 ## values of this field are @qcode{"nofill"} and @qcode{"ict"}.  The
 ## @qcode{"nofill"} variant performs incomplete Cholesky with zero-fill
-## [IC(0)].  The @qcode{"ict"} variant performs incomplete Cholesky with
-## threshold dropping [ICT].  The default value is @qcode{"nofill"}.
+## @nospell{[IC(0)]}.  The @qcode{"ict"} variant performs incomplete Cholesky
+## with threshold dropping @nospell{[ICT]}.  The default value is
+## @qcode{"nofill"}.
 ##
 ## @item droptol
 ## Drop tolerance when type is @qcode{"ict"}.
-## Non-negative scalar used as a drop tolerance when performing ICT@.  Elements
-## which are smaller in magnitude than a local drop tolerance are dropped from
-## the resulting factor except for the diagonal element which is never dropped.
+## Non-negative scalar used as a drop tolerance when performing @nospell{ICT}@.
+## Elements which are smaller in magnitude than a local drop tolerance are
+## dropped from the resulting factor except for the diagonal element which is
+## never dropped.
 ## The local drop tolerance at step j of the factorization is
 ## @code{norm (@var{A}(j:end, j), 1) * droptol}.  @code{droptol} is ignored if
 ## @code{type} is @qcode{"nofill"}.  The default value is 0.
 ##
 ## @item michol
 ## Indicate whether modified incomplete Cholesky [MIC] is performed.
 ## The field may be @qcode{"on"} or @qcode{"off"}.  When performing MIC, the
 ## diagonal is compensated for dropped elements to enforce the relationship
@@ -130,17 +132,17 @@
 ## norm (A - L * L', "fro") / norm (A, "fro")
 ## ans =  0.062327
 ## @end group
 ## @end example
 ##
 ## References for implemented algorithms:
 ##
 ## [1] @nospell{Y. Saad}. "Preconditioning Techniques." @cite{Iterative
-## Methods for Sparse Linear Systems}, PWS Publishing Company, 1996.
+## Methods for Sparse Linear Systems}, @nospell{PWS} Publishing Company, 1996.
 ##
 ## [2] @nospell{M. Jones, P. Plassmann}: @cite{An Improved Incomplete
 ## Cholesky Factorization}, 1992.
 ## @seealso{chol, ilu, pcg}
 ## @end deftypefn
 
 function L = ichol (A, opts = struct ())
 
diff --git a/scripts/sparse/ilu.m b/scripts/sparse/ilu.m
--- a/scripts/sparse/ilu.m
+++ b/scripts/sparse/ilu.m
@@ -40,27 +40,27 @@
 ##
 ## @table @asis
 ## @item @qcode{"nofill"}
 ## Perform ILU factorization with 0 level of fill in, known as ILU(0).  With
 ## type set to @qcode{"nofill"}, only the @code{milu} option is used; all other
 ## fields are ignored.
 ##
 ## @item @qcode{"crout"}
-## Perform the Crout version of ILU factorization, known as ILUC@.  With type
-## set to @qcode{crout}, only the @code{droptol} and @code{milu} options are
-## used; all other fields are ignored.
+## Perform the @nospell{Crout} version of ILU factorization, known as
+## @nospell{ILUC}@.  With type set to @qcode{crout}, only the @code{droptol}
+## and @code{milu} options are used; all other fields are ignored.
 ##
 ## @item @qcode{"ilutp"}
 ## (default) Performs ILU factorization with threshold and pivoting.
 ## @end table
 ##
-## If type is not specified, the ILU factorization with pivoting ILUTP is
-## performed.  Pivoting is never performed with type set to @qcode{"nofill"} or
-## @qcode{"crout"}.
+## If type is not specified, the ILU factorization with pivoting @nospell{ILUTP}
+## is performed.  Pivoting is never performed with type set to @qcode{"nofill"}
+## or @qcode{"crout"}.
 ##
 ## @item droptol
 ## Drop tolerance of the incomplete LU factorization.  @code{droptol} is a
 ## non-negative scalar.  The default value is 0, which produces the complete
 ## LU factorization.
 ##
 ## The nonzero entries of @var{U} satisfy
 ##
diff --git a/scripts/statistics/base/lscov.m b/scripts/statistics/base/lscov.m
--- a/scripts/statistics/base/lscov.m
+++ b/scripts/statistics/base/lscov.m
@@ -37,18 +37,18 @@
 ## is currently ignored.
 ##
 ## Besides the least-squares estimate matrix @var{x} (p-by-k), the
 ## function also returns @var{stdx} (p-by-k), the error standard
 ## deviation of estimated @var{x}; @var{mse} (k-by-1), the estimated
 ## data error covariance scale factors (@math{\sigma^2}); and @var{S}
 ## (p-by-p, or p-by-p-by-k if k > 1), the error covariance of @var{x}.
 ##
-## Reference: Golub and Van Loan (1996), Matrix Computations (3rd Ed.), Johns
-## Hopkins, Section 5.6.3
+## Reference: @nospell{Golub and Van Loan} (1996),
+## Matrix Computations (3rd Ed.), Johns Hopkins, Section 5.6.3
 ##
 ## @end deftypefn
 ## @seealso{ols, gls, lsqnonneg}
 
 ## Author: Nir Krakauer
 
 function [x, stdx, mse, S] = lscov (A, b, V = [], alg)
 
